spmd/tensor/api.py

```diff
@@ -234,3 +275,8 @@ def device_mesh(self) -> DeviceMesh:
         # caller who want a different device_mesh
         # should call redistribute instead.
         return self._device_mesh
+
+    def contiguous(self) -> "Tensor":
```

Comment:
can we add `_view_with_sharding_dim_change` and this one to the tp ops file as well? Let me add a `__torch_function__` method and you can register these two ops in the op table.